Havit/Havit/Global/UIComponent/ContentsCollectionViewCell.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ContentsCollectionViewCell.swift
+//  Havit
+//
+//  Created by 박예빈 on 2022/01/12.
+//
+
+import UIKit
+import SnapKit
+import RxSwift
+
+class ContentsCollectionViewCell: UICollectionViewCell {
```

Comment:
P4;
```suggestion
final class ContentsCollectionViewCell: UICollectionViewCell {
```
앞에 final 키워드를 넣어보는건 어떨까요? 
ContentsCollectionViewCell이 상속, 상속에 따른 재정의를 하지 않을거 같다면 final를 사용해서 상속을 막을 수 있습니다. 
https://github.com/apple/swift/blob/main/docs/OptimizationTips.rst#advice-use-final-when-you-know-the-declaration-does-not-need-to-be-overridden

Reply:
final 키워드 저학년때 자바에서 배웠던 기억이.. 다 까먹었지만... 일종의 상수 역할과 비슷하네요!